tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert items == []
```

Comment:
👍 Comparing to a literal empty list both ensures that the result is an actual list, and that it is actually empty. To confirm the behavior of the function, those are both good to check, so is preferable to a falsy-style check such as `assert not items`.